ros2bag_image_extractor.py

```diff
@@ -79,27 +88,49 @@ def undistort(input, distortion_data):
 # ---------------------------------------------------------------------------- #
 
 # ---------------- Create reader instance and open for reading --------------- #
-with Reader(ROSBAG_FILE) as reader:
-
-    camera_found = False
-
-    for connection in reader.connections:
-        if args.verbose:
-            print(connection.topic," : ", connection.msgtype)
-        if 'camera' in connection.topic:
-            camera_found = True
-
-    if camera_found is False:
-        exit()
-
-    # Iterator dictionary to go over camera messages
+# with Reader(ROSBAG_FILE) as reader:
```

Comment:
Delete instead of commenting as much as possible. This is a git repo, can be recovered later if really needed.